set.mm

```diff
@@ -417512,33 +417512,33 @@ or axiom of dependent choice ( ~ ax-dc ) can be used instead.
        and moving them up to the ~ ax-4 through ~ ax-9 section.
        </li>
 
-       <li><b>Alternative (ALT) proofs.</b>
-       If a different proof is significantly shorter or clearer but
-       uses more or stronger axioms, we prefer to make that proof an
-       "alternative" proof (marked with an ALT label suffix), even if
-       this alternative proof was formalized first.
+       <li><b>Alternate (ALT) proofs.</b>
```

Comment:
I am not sure whether alternate is a good replacement for alternative.  I thought 'alternate' means 'repeatedly changing' as in alternate current or alternate number sequence (1, -1, 1, -1...).  

This is from the Oxford dictionary:
1. every other; every second.
"she was asked to attend on alternate days"
2. North American
another term for alternative.
"a novel set in an alternate universe"

I think here is support from native English speakers called for.  I personally tend towards 'alternative'.

Reply:
I see (for the adjectives):
* https://www.dictionary.com/browse/alternate : constituting an alternative (e.g., "an alternate route")
* https://www.dictionary.com/browse/alternative : there is not quite such meaning

and when interchangeable, I tend to prefer the shorter alternative (<-- this is the noun "alternative" !).

Reply:
I entered 'alternate vs alternative' into Google and the first hit is
https://www.merriam-webster.com/words-at-play/differences-between-alternate-and-alternative
which elaborates on the question at hand.  There seems to be no distinct favorite:
``
Usage commentators, however, have often emphasized a distinction: one should use alternate to mean "occurring or succeeding by turns" and alternative to mean "offering or expressing a choice." Yet a look at the way these words are used makes it clear that things aren't quite so simple.
``
IMO the version of the original author wins in such cases, otherwise I imagine a future of frequently alternating versions here and elsewhere for whatever liking is dominating at a particular moment.

My English skills are not sufficient to decide on this.  So I am going to accept any solution.